.circleci/config.yml

```diff
@@ -112,6 +112,26 @@ jobs:
       - store_artifacts:
           path: /tmp/artifacts
 
+  server_help:
```

Comment:
Do we really need to add this to CircleCI check that runs for every `docs` commit?